File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/service/message/reactiveobject/BrowseMemberResponse.java
package org.opcfoundation.webserver.service.message.reactiveobject;

import org.opcfoundation.webserver.types.common.UaReferenceDescriptor;

import java.util.List;

public class BrowseMemberResponse {
    private final List<UaReferenceDescriptor> children;

    public BrowseMemberResponse(List<UaReferenceDescriptor> children)
    {
        this.children = children;
    }

    public List<UaReferenceDescriptor> getChildren() {
        return children;
    }
}

File: opcua-webservice-node/source/src/server/service/message/reactiveobject/BrowseMemberResponse.ts
import { UaReferenceDescriptor } from "../../../types";

export class BrowseMemberResponse {
    private readonly _children: Array<UaReferenceDescriptor>;

    constructor(children: Array<UaReferenceDescriptor>) {
        this._children = children;
    }

    get children(): Array<UaReferenceDescriptor> {
        return this._children;
    }
}